### PR TITLE
Fix typo in Banishing Smite description

### PR DESCRIPTION
**Pull request by 153957**

### CODE REVIEW

File: Spells/Player's Handbook.xml [hunk 1]
```diff
@@ -406,7 +406,7 @@
...
+		<text>The next time you hit a creature with a weapon attack before this spell ends, your weapon crackles with force, and the attack deals an extra 5d10 force damage to the target. Additionally, if this attack reduces the target to 50 hit points or fewer, you banish it. If the target is native to a different plane of existence than the one you're on, the target disappears, returning to its home plane. If the target is native to the plane you're on, the creature vanishes into a harmless demiplane. While there, the target is incapacitated. It remains there until the spell ends, at which point the tart reappears in the space it left or in the nearest unoccupied space if that space is occupied.</text>
```
Comment (Cphrampus) [inline Spells/Player's Handbook.xml]:
```suggestion
		<text>The next time you hit a creature with a weapon attack before this spell ends, your weapon crackles with force, and the attack deals an extra 5d10 force damage to the target. Additionally, if this attack reduces the target to 50 hit points or fewer, you banish it. If the target is native to a different plane of existence than the one you're on, the target disappears, returning to its home plane. If the target is native to the plane you're on, the creature vanishes into a harmless demiplane. While there, the target is incapacitated. It remains there until the spell ends, at which point the target reappears in the space it left or in the nearest unoccupied space if that space is occupied.</text>
```

Comment (Cphrampus) [reply Spells/Player's Handbook.xml]: In case that isn't highlighting well, there was an instance of tart rather than target

**Comment by 153957** [comment]
The same typo exists in the pre generated Half-Elf Bard 10. Should I also fix the description there or can you somehow programmatically generate the pregens from the current compendium?

**Comment by Cphrampus** [comment]
I'm sorry, I thought I replied to this. If you could also fix that, that would be great. As for programmatically generating the files, it may be possible, I just built them in the app and exported them so that they were available. At the very least, it may be possible to make a version that references the spells so that the app can populate them for the specific character when imported, I will have to look at that

**Pull request merged**